AnnotatoBackend/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.5
+import PackageDescription
+
+let package = Package(
+    name: "AnnotatoBackend",
+    platforms: [
+       .macOS(.v12)
```

Comment:
Should we use the v10_15 version of Vapor to be more flexible? I haven't updated to macos 12 and will not be able to run the app

Reply:
To use the v10_15 version, use the flag `--branch macos10-15` when creating the Vapor project - see [here](https://docs.vapor.codes/4.0/hello-world/#:~:text=Vapor%20and%20the%20template%20now%20uses%20async/await%20by%20default.%20If%20you%20cannot%20update%20to%20macOS%2012%20and/or%20need%20to%20continue%20to%20use%20EventLoopFutures%2C%20use%20flag%20%2D%2Dbranch%20macos10%2D15.)